src/Component.php

```diff
@@ -75,6 +75,11 @@ public function getUpdatesQueryString()
     {
         return $this->updatesQueryString;
     }
+    
+    public function mapStateToUrl()
+    {
+    	return null;
+    }
```

Comment:
We could get a performance gain if we also include the props that affect the URL in this response, but I feel like the API that I would want is just return a string and have livewire work.

One solution would be to add an optional property to the component, something like `$urlProps`, that could be used to filter down the props that are associated with the url state (this is the pattern that React uses for memoizing hooks).